pybaseball/statcast.py

```diff
@@ -61,7 +61,11 @@ def large_request(start_dt,end_dt,d1,d2,step,verbose):
         # if these dates are detected, check if the next season is within the user's query
         # if yes, fast-forward to the next season to avoid empty requests
         # if no, break the loop. all useful data has been pulled.
-        if ((d.month < 4 and d.day < 15) or (d1.month > 10 and d1.day > 14)):
+        if d.month < 4 and d.day < 15:
```

Comment:
This would potentially miss some dates. E.g., 2/28.

```suggestion
        if (d.month == 3 and d.day < 15) or d.month <= 2:
```